walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,71 @@
 
 package com.google.sps;
 
+import static java.util.Collections.EMPTY_LIST;
+import static java.util.Collections.disjoint;
+import static java.util.Collections.sort;
+import static java.util.Arrays.asList;
+
+import java.util.Comparator;
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.List;
+
 
+// FindMeetingQuery finds open meeting timeslots throughout the day.
 public final class FindMeetingQuery {
+  /** 
+  *  @param events Set of events that attendees have, that need to be avoided.
+  *  @param request The duration of requested meeting and people attending. 
+  *  @return A list of open meeting timeslots.  
+  */
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
```

Comment:
optional/nit: Maybe the caller can have something like FindMeetings.calculateTimeSlots()?  Up to you.  If you want to do it, does not need to be done in this CL.  Right now it's FindMeetingQuery.query() so you're saying "query" twice but not really saying what it's returning.

Reply:
Noted.  Will add to a future CL.